Cargo.toml

```diff
@@ -567,6 +567,7 @@ move-abstract-stack = { path = "external-crates/move/crates/move-abstract-stack"
 fastcrypto = { git = "https://github.com/MystenLabs/fastcrypto", rev = "c101a5176799db3eb9c801b844e7add92153d291" }
 fastcrypto-tbls = { git = "https://github.com/MystenLabs/fastcrypto", rev = "c101a5176799db3eb9c801b844e7add92153d291" }
 fastcrypto-zkp = { git = "https://github.com/MystenLabs/fastcrypto", rev = "c101a5176799db3eb9c801b844e7add92153d291", package = "fastcrypto-zkp" }
+fastcrypto-vdf = { git = "https://github.com/MystenLabs/fastcrypto", rev = "c101a5176799db3eb9c801b844e7add92153d291", features = ["experimental"] }
```

Comment:
this enables `experimental` for all fastcrypto, in Sui

Reply:
That's not true. The `experimental` feature enabled here is defined on `fastcrypto-vdf` and does not enable other features. This can be verified by running `cargo tree -e features -i fastcrypto` which lists all features enabled on the `fastcrypto` dependency in Sui and where they are enabled. In this case, only the `default`, `copy_key` and `aes` are enabled.

I can see that the naming is a bit confusing though, and if the `experimental` feature in `fastcrypto-vdf` enabled the `experimental` in other fastcrypto sub-crates, you'd be correct.